app/services/search/google/search_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Search
+  module Google
+    class SearchService < Search::BaseService
+      GOOGLE_SEARCH_BASE_URL = 'https://www.google.com/search'
+
+      def initialize(keyword, language = 'en')
+        super(keyword, language)
+      end
+
+      def call
+        uri = URI.parse("#{GOOGLE_SEARCH_BASE_URL}?q=#{keyword}&hl=#{language}&lr=#{language}")
```

Comment:
As you're using a base service, should we default a method `search_url` that uses `BASE_URL` constant and gets the corresponding value?

Reply:
@hoangmirs The base search service has been refactored 🚀